.github/workflows/dotnet.yml

```
@@ -22,7 +22,6 @@ jobs:
       uses: actions/setup-dotnet@v3
       with:
         dotnet-version: 6.0.x
```

The latest `dotnet.yml` format implies specifying .NET version as `'6.0.x'` instead of just `6.0.x`:
```suggestion
        dotnet-version: '6.0.x'
```

---

Fixed